src/ffi/bindings/orx_bindings.ml

```diff
@@ -1291,6 +1314,8 @@ module Bindings (F : Ctypes.FOREIGN) = struct
         | Shader : (Shader_event.t, Shader_event.payload) t
         | Sound : (Sound_event.t, Sound_event.payload) t
         | Time_line : (Time_line_event.t, Time_line_event.payload) t
+		| Animation : (Anim_event.t, 
```

Comment:
Style nit: all of these `Animation` additions can be moved to the top of each type definition and `match` to keep order consistent.